src/operator/controllers/intents_reconcilers/exp/database.go

```diff
@@ -68,8 +74,16 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	if err := r.otterizeClient.ApplyDatabaseIntent(ctx, intentInputList, action); err != nil {
+		errType, errMsg, ok := graphqlclient.GetGraphQLUserError(err)
+		if !ok || errType != graphqlclient.UserErrorTypeAppliedIntentsError {
+			r.RecordWarningEventf(intents, ReasonApplyingDatabaseIntentsFailed, "Failed applying database intents")
+			return ctrl.Result{}, err
+		}
+		r.RecordWarningEventf(intents, ReasonApplyingDatabaseIntentsFailed, fmt.Sprintf("%s", errMsg))
 		return ctrl.Result{}, err
 	}
 
+	r.RecordNormalEventf(intents, ReasonAppliedDatabaseIntents, "Successfully applied database intents")
```

Comment:
How many intents? This is different from other reconcilers, they show a message with the number of intents successfully reconciled.